api/src/main/java/io/druid/guice/Binders.java

```diff
@@ -22,16 +22,18 @@
 import com.google.inject.Binder;
 import com.google.inject.Key;
 import com.google.inject.multibindings.MapBinder;
+import io.druid.guice.annotations.PublicApi;
 import io.druid.segment.loading.DataSegmentArchiver;
 import io.druid.segment.loading.DataSegmentFinder;
-import io.druid.segment.loading.DataSegmentMover;
 import io.druid.segment.loading.DataSegmentKiller;
+import io.druid.segment.loading.DataSegmentMover;
 import io.druid.segment.loading.DataSegmentPuller;
 import io.druid.segment.loading.DataSegmentPusher;
 import io.druid.tasklogs.TaskLogs;
 
 /**
  */
+@PublicApi
```

Comment:
actually its more of an utility class and shouldn't necessarily be directly used in custom user extension (it can be used in out-of-the-box extensions) and those should really just use MapBinder.xxx .